test/plugs/parse_slack_payload_test.exs

```diff
@@ -0,0 +1,21 @@
+defmodule EspiDni.Plugs.ParseSlackPayloadTest do
+
+  use EspiDni.ConnCase, async: true
+
+  test "does nothing if the payload is missing" do
+    conn =
+      conn(:get, "/", %{})
+      |> EspiDni.Plugs.ParseSlackPayload.call(%{})
+
+    assert get_in(conn.assigns, [:payload, "token"]) == nil
+    refute conn.halted
+  end
+
+  test "parses and sets the payload" do
+    conn =
+      conn(:get, "/", %{payload: "{\"foo\": \"bar\" }"})
```

Comment:
Would it make sense to encode it as json instead of creating the json string manually? Sth. like

```elixir
Poison.encode!(%{"foo" => "bar"})
```
?